### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by krikic**
Dragon Ball Fusion game upload images and info game

### CODE REVIEW

File: Dragon Ball Fusions/game.dat [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title = "Dragon Ball Fusions"
+description = ""Dragon Ball Fusions (ドラゴンボールフュージョンズ) is a
```
Comment (chris062689) [inline Dragon Ball Fusions/game.dat]: Please remove the extra ", and make sure this is all on a single line.

File: Dragon Ball Fusions/game.dat [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+github_issues = [1654]
+needs_system_files = false
+needs_shared_font = false
+
+[[ releases ]]
+title = "0004000000125600"
+region = "EUR"
+release_date = "2017-02-17"
+
+
```
Comment (chris062689) [inline Dragon Ball Fusions/game.dat]: Remove this extra line.

**Comment by chris062689** [comment]
Please also correct the issues in the build log.

**Comment by j-selby** [comment]
Superseded by #27. Next time, don't spam PRs. Thanks.

**Pull request closed without merging**